src/pj_obs_api.c

```diff
@@ -441,10 +524,14 @@ void *proj_release (void *buffer) {
 double proj_torad (double angle_in_degrees) { return PJ_TORAD (angle_in_degrees);}
 double proj_todeg (double angle_in_radians) { return PJ_TODEG (angle_in_radians);}
 
+int proj_has_inverse(PJ *P) {
+    return (P->inv != 0 || P->inv3d != 0 || P->invobs != 0);
+}
 
-/* The shape of jazz to come! */
+double proj_dmstor(const char *is, char **rs) {
+    return dmstor(is, rs);
```

Comment:
At first I think we should agree on what's going to be deprecated, if anything at all.